assets/js/components/DetailsPermaLinks.js

```diff
@@ -49,7 +49,11 @@ export default function DetailsPermaLinks( { title, path, serviceURL } ) {
 
 	return (
 		<Fragment>
-			<Link href={ serviceURL || detailsURL } external={ !! serviceURL }>
+			<Link
+				href={ serviceURL || detailsURL }
+				external={ !! serviceURL }
+				hideExternalIndicator
```

Comment:
I'm not sure I'm onboard with this change as it may be problematic for accessibility reasons? Thoughts @tofumatt?

Marking external links as such is an a11y best practice, and I can't imagine that Material 3 would dictate otherwise.

Reply:
@felixarntz We are only hiding the icon for some external links. I don't think it'll cause any accessibility issues since the `aria-label` states that the link will open in a new tab. Hiding the external icon is something that UX requested for links within tables.